src/core/Akka/Actor/ActorPath.cs

```diff
@@ -298,10 +297,14 @@ private static bool TryParseAddress(string path, out Address address, out Uri ur
             //This code corresponds to AddressFromURIString.unapply
             uri = null;
             address = null;
-
-            if (!Uri.TryCreate(path, UriKind.Absolute, out uri))
+            try
```

Comment:
Looks like I have some rebase version issue, that older code somehow got in here